### ISSUE TITLE
SIGWINCH is not passed to tmux

### ISSUE THREAD

**Issue by huiyiqun** [elves/elvish#287]
There used to be a similar [issue](https://github.com/fish-shell/fish-shell/issues/3180) in fish.

This is how I start tmux:
```
use utils

fn start {
    if ==s $E:TMUX ''; then
        if utils:ok { tmux ls }; then
            # There are existing sessions
            exec tmux attach
        else
            exec tmux
        fi
    fi
}
```

If the terminal is resized suddenly, the tmux seems not receiving SIGWINCH from the terminal.

**Comment by huiyiqun** [comment]
In opposite to fish, if I remove the `exec`, tmux receive SIGWINCH correctly.

Mabye problem of `exec`?

**Comment by huiyiqun** [comment]
I'm sure that elvish didn't set tty to the correct mode when `exec` a fullscreen program like
[musicbox](https://github.com/darknessomi/musicbox)

**Comment by huiyiqun** [comment]
> I'm sure that elvish didn't set controlling tty correctly when exec a fullscreen program like
musicbox

It seems different issue for that `musicbox` is not foreground process when `exec`ed.

Let's discuss it in #288.

**Comment by huiyiqun** [comment]
Because `exec`ed process is not in foreground group.

**Comment by xiaq** [comment]
Has this been fixed with 36271874d8ddb2519739aa431564e857b5a52945?
